youtube_downloader/downloader.py

```
@@ -178,43 +179,38 @@ def download(self, output_file='video.mp4', itag=None, quality=None):
                         self._apply_proxy(maybe)
-                # Record against the actual active proxy
                 current_proxy = self._active_proxy
                 
                 response = self.session.get(selected['url'], headers=headers, stream=True, timeout=60)
-                
-                # Check for rate limiting
+               
                 if response.status_code == 429:
                     if self.proxy_manager and attempt < retries - 1:
-                        print("\n⚠ Rate limited during download. Rotating proxy...")
+                        print("\nâš  Rate limited during download. Rotating proxy...")
```

Ruff (0.14.1)

String contains ambiguous ` ` (NO-BREAK SPACE). Did you mean ` ` (SPACE)?

(RUF001)